airflow/api/common/proto/variable.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto2";
+
+package airflow;
+
+message Variable {
```

Comment:
Why are we using protobufs? What is the implication of adding new fields to the Variables table with respect to the IDL and API version?